cmd/lotus-miner/actor.go

```diff
@@ -18,15 +22,17 @@ import (
 	rlepluslazy "github.com/filecoin-project/go-bitfield/rle"
 	"github.com/filecoin-project/go-state-types/abi"
 	"github.com/filecoin-project/go-state-types/big"
-	"github.com/filecoin-project/go-state-types/builtin"
+	builtint "github.com/filecoin-project/go-state-types/builtin"
```

Comment:
Let's keep this with the original import name so we're not changing a bunch of other stuff in here.